package.json

```diff
@@ -10,7 +10,10 @@
   },
   "workspaces": [
     "eslint-configs/*",
-    "packages/*"
+    "packages/internal-test-env",
+    "packages/internal-playwright-testids",
+    "packages/internal-playwright-helpers",
+    "packages/jest-jsdom-polyfills"
```

Comment:
This change shouldn't be needed, this just tells npm to glob that directory..

Reply:
I'll try what you suggested in your comment and have lerna run the build, because I actually thought that was already the case and that lerna was hooking into the workspaces setup. This changes just adds an order in which the packages are built, because using alphabetical order results in `packages/internal-playwright-helpers` being built first, whil it has dependencies on the other packages.